dapps/valuetransfers/packages/tangle/tangle.go

```diff
@@ -1324,16 +1369,18 @@ func (tangle *Tangle) bookTransaction(cachedTransaction *transaction.CachedTrans
 		return true
 	})
 
-	// fork the conflicting transactions into their own branch
-	for consumerID, conflictingInputs := range conflictingInputsOfFirstConsumers {
-		_, decisionFinalized, forkedErr := tangle.Fork(consumerID, conflictingInputs)
-		if forkedErr != nil {
-			err = forkedErr
+	// fork the conflicting transactions into their own branch if a decision is still pending
+	if decisionPending = !finalizedConflictingSpenderFound; decisionPending {
```

Comment:
```suggestion
	decisionPending = !finalizedConflictingSpenderFound
	if decisionPending {
```
Nothing is declared in the short statement, so there is really no point to have this in the condition